### ISSUE TITLE
[BUG] Message: SyntaxError("(unicode error) 'utf-8' codec can't decode)

### ISSUE THREAD

**Issue by tbroglie** [dashingsoft/pyarmor#1006]
Hi there ! 

**My config**
Python 3.9.13
Pyarmor 8.0.10 (pro), 00xxxx, non-profits

**My issue**
I am trying to use the bcc or rft modes, with a pro license. 
I have executed the following command on the foo.py example.

`pyarmor gen --enable-rft foo.py
`

```
Message: SyntaxError("(unicode error) 'utf-8' codec can't decode byte 0xa5 in position 0: invalid start byte", ('<rftmaker>', 1, 5, '��Ȋ�\x0f�����\x0e�\n'))
Arguments: ()
```

It also makes this with bcc mode. 

**Resolve tried**
I have added the following code at the beginning of foo.py which did not help.
```
#! /usr/bin/env python
# -*- coding: utf-8 -*-
```

The full traceback is the following : 
```
INFO     Python 3.9.13
INFO     Pyarmor 8.0.10 (pro), 00xxxx, non-profits
INFO     search inputs ...
INFO     find script diptyque-exec/test_pyarmor.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
INFO     write dist\pyarmor_runtime_00xxxx\pyarmor_runtime.pyd
INFO     patch runtime file
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "test_pyarmor"
INFO     obfuscating file test_pyarmor.py
CRITICAL unknown error, please check pyarmor.error.log
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\__main__.py", line 559, in main
    main_entry(sys.argv[1:])
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\__main__.py", line 547, in main_entry
    args.func(ctx, args)
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\__main__.py", line 144, in cmd_gen
    builder.process(options, pack=args.pack)
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\generate.py", line 153, in process
    self._obfuscate_scripts()
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\generate.py", line 118, in _obfuscate_scripts
    code = Pytransform3.generate_obfuscated_script(self.ctx, r)
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\core\__init__.py", line 48, in generate_obfuscated_script
    return m.generate_obfuscated_script(ctx, res)
  File "<maker>", line 713, in generate_obfuscated_script
  File "<maker>", line 488, in process
  File "<rftmaker>", line 1
    ��Ȋ�������
        ^
SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0xa5 in position 0: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\ThomasBreuille\miniconda3\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
  File "C:\Users\ThomasBreuille\miniconda3\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 1149-1152: character maps to <undefined>
Call stack:
  File "C:\Users\ThomasBreuille\miniconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\ThomasBreuille\miniconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\ThomasBreuille\miniconda3\Scripts\pyarmor.exe\__main__.py", line 7, in <module>
    sys.exit(main_entry_8())
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\pyarmor.py", line 1632, in main_entry_8
    call_pyarmor_cli()
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\pyarmor.py", line 1617, in call_pyarmor_cli
    main()
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\__main__.py", line 565, in main
    log_exception(e)
  File "C:\Users\ThomasBreuille\miniconda3\lib\site-packages\pyarmor\cli\__main__.py", line 504, in log_exception
    log.exception(e)
Message: SyntaxError("(unicode error) 'utf-8' codec can't decode byte 0xa5 in position 0: invalid start byte", ('<rftmaker>', 1, 5, '��Ȋ�\x0f�����\x0e�\n'))
Arguments: ()
ERROR    (unicode error) 'utf-8' codec can't decode byte 0xa5 in position 0: invalid start byte (<rftmaker>, line 1)
```

By the way, pyarmor.error.log is empty.

Thanks,

Thomas je

**Comment by jondy** [comment]
Got it, thanks. I'll check it.

**Comment by jondy** [comment]
Fixed in 8.0.11
